packages/batch-submitter/README.md

```diff
@@ -1,3 +1,4 @@
+[![codecov](https://codecov.io/gh/ethereum-optimism/optimism/branch/master/graph/badge.svg?token=0VTG7PG7YR&flag=batch-submitter)](https://codecov.io/gh/ethereum-optimism/optimism)
```

Comment:
As per this url, it should show the badges as per the coverage on master in the readme, but it doesn't quite exist yet. If it would be helpful, I can change this branch for the time being so results in are viewable in the `README`s of this PR